src/components/StatusSquare/index.tsx

```diff
@@ -24,6 +24,7 @@ export const mapVariantTaskStatusToColor = {
   [TaskStatus.SystemUnresponsive]: failurePurple,
   [TaskStatus.SetupFailed]: failureLavender,
   [TaskStatus.StatusBlocked]: gray.dark1,
+  [TaskStatus.Aborted]: gray.light1,
```

Comment:
Do we want to add this to mapVariantTaskStatusToDarkColor as well?

Reply:
Thanks forgot about that just updated!